ci/run_cpp.sh

```diff
@@ -17,17 +17,11 @@ run_cpp_tests() {
   RUNTIME_PATH=${CONDA_PREFIX:-./}
   BINARY_PATH=${RUNTIME_PATH}/bin
 
-  # Disable memory get/put with RMM in protov1, it always segfaults.
-  CMD_LINE="timeout 10m ${BINARY_PATH}/gtests/libucxx/UCXX_TEST --gtest_filter=-*RMM*Memory*"
-
-  log_command "${CMD_LINE}"
-  UCX_TCP_CM_REUSEADDR=y ${CMD_LINE}
-
   # Only test memory get/put with RMM in protov2, as protov1 segfaults.
```

Comment:
Do we still need this comment about protov1?

Also, where is the proto version being set? I see `UCX_PROTO_ENABLE=y` was removed below. What does that mean?